frappe/www/printview.py

```diff
@@ -254,8 +254,14 @@ def get_print_format(doctype, print_format):
 		elif print_format.raw_commands and print_format.raw_printing:
 			return print_format.raw_commands
 		else:
-			frappe.throw(_("No template found at path: {0}").format(path),
```

Comment:
This is kind of a breaking change. This method throws an error when print_format template is not found but after your change it doesn't. You should handle this at client side. Catch this error and then show the appropriate HTML.

Reply:
aight

Reply:
update?

Reply:
Need to clarify with Faris, working on it